package.json

```diff
@@ -56,7 +56,7 @@
     "eslint-plugin-lit": "^1",
     "eslint-plugin-sort-class-members": "^1",
     "frau-ci": "^1",
-    "karma-sauce-launcher": "^4",
+    "karma-sauce-launcher": "^2",
```

Comment:
Version 4 is causing a failure `TypeError: Cannot read property 'title' of undefined` with legacy edge being added